pulser/tests/test_devices.py

```diff
@@ -43,10 +45,12 @@ def test_init():
 
 def test_mock():
     dev = pulser.devices.MockDevice
-    assert dev.dimensions == 3
+    assert dev.dimensions == 2
```

Comment:
It's actually `dev.dimensions == 3` now, this is here because your branch way outdated with respect to `develop`.

Reply:
Oups sorry